compiler/optimizer/OMROptimizer.hpp

```diff
@@ -114,7 +114,8 @@ enum
    IfEnabledAndMoreThanOneBlock,
    IfEnabledAndMoreThanOneBlockMarkLastRun,
    IfAOTAndEnabled,
-   IfMethodHandleInvokes, // JSR292: Extra analysis required to optimize MethodHandle.invoke
+   IfMethodHandleInvokes, //JSR292: Extra analysis required to optimize MethodHandle.invoke
+   IfMethodHandleInvokesNoDebug, // JSR292 optimizations that should only apply in non-debug mode
```

Comment:
There is a convention of having an And in the name when there is a union of two conditions - eg AOTAndEnabled. Could you please add the And in?